ai_genomics/getters/gtr.py

```diff
@@ -57,4 +56,4 @@ def get_ai_genomics_project_table(local: bool = True) -> pd.DataFrame:
         return pd.read_csv(GTR_OUTPUTS_DIR / GTR_PROJ_NAME)
 
     else:
-        return fetch_s3(f"outputs/gtr/{GTR_PROJ_NAME}")
+        return load_s3_data("ai-genomics", f"outputs/gtr/{GTR_PROJ_NAME}")
```

Comment:
you can just use BUCKET_NAME as you've imported it instead of "ai-genomics" but its much of a muchness as this works too